assets/css/climatehub.css

```diff
@@ -1,8 +1,7 @@
 
 /*FOOTER*/
 
-/*climate hub logo*/
-.chlogo {
+.footer-logo {
   width: 70%;

```

Comment:
I should mentioned this earlier, there's a bootstrap class you can use for this. In this case it would be `w-75` for a width = 75%. If 75% is too much you can leave it at 70.

Reply:
will change it in next commit